### ISSUE TITLE
psychmeta support 

### ISSUE THREAD

**Issue by bwiernik** [malcolmbarrett/tidymeta#6]
This is a really cool package. We would love to be able to have compatibility between your package and psychmeta (https://github.com/psychmeta/psychmeta). psychmeta natively produces model results in tibble format. If you are interested, we would love to help to make your plot geoms compatible with psychmeta objects.

**Comment by malcolmbarrett** [comment]
Yes, that would be great. It's always been my intention to support multiple packages. I will warn you that tidymeta is undergoing some structural changes over the next few weeks, but I suspect that the plotting functions will be more or less consistent with how they are now
